content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md

```diff
@@ -0,0 +1,62 @@
+---
+date: '2023-01-26'
+title: "Corda 5.0 Beta 1.1 Release Notes"
+menu:
+  corda-5-beta:
+    parent: corda-5-beta-release-notes
+    identifier: corda-5-beta-release-notes-1.1
+    weight: 1000
+section_menu: corda-5-beta
+---
+
+Corda 5.0 Beta is a pre-release version for testing purposes only.
+{{< note >}}
+If you are not part of the current beta program, the Corda 5.0 Beta documentation is for information only.
+Please contact R3 if you are interested in joining the Beta program.
+{{< /note >}}
+
+## Enhancements
+
+This section describes the new features in Corda 5.0 Beta 1.1.
+
+### CSDE 
+
+#### Beta 1.1 API
+
+The [CorDapp Standard Development Environment (CSDE)](cordapp-standard-development-environment/csde.html) has been upgraded to use the Corda 5.0 Beta 1.1 API. 
+
+#### Example UTXO Ledger CorDapp
+
+The CSDE now includes an [example  UTXO (unspent transaction output) ledger CorDapp](....). This simple chat application  is available in both the [CSDE-cordapp-template-kotlin](https://github.com/corda/CSDE-cordapp-template-kotlin) and [CSDE-cordapp-template-java](https://github.com/corda/CSDE-cordapp-template-java) repositories.
+
+#### Gradle Helpers
+
+The CSDE gradle helpers have been renamed and grouped into new directories to improve usability.
+
+## Resolved Issues
+
+This section describes the issues resolved in Corda 5.0 Beta 1.1.
+
+### UTXO Ledger
+
+#### Transaction Failures
+Input states to a UTXO transaction were marked as consumed when the transaction was first persisted, before it was completely signed and notarised. 
+As a result, if counter-signing or notarisation failed, states were incorrectly marked as consumed in the initiating's node vault and were no longer available as inputs.
+
+#### State Relevancy Flag
+
+Transactions were finalized differently by the `UtxoReceivedFinalityFlow` and `UtxoFinalityFlow` causing the relevancy flag to be inconsistently set. 
+As a result, for the initiating participant on a transaction, a Relevant State could incorrectly be flagged as available.
+As of this release, both `UtxoReceivedFinalityFlow` on the counterparty side and `UtxoFinalityFlow` on the initiating side correctly persist State relevancy when the transaction is verified. 
+
+#### ContractState Cast Exception
+
+An exception was incorrectly thrown if an attempt was made to cast a `ContractState`.
```

Comment:
An exception was incorrectly thrown if an attempt was made to call `getInputStateAndRefs(…)`.

Reply:
Thanks!